### ISSUE TITLE
Username Password

### ISSUE THREAD

**Issue by esyekas** [melentye/flink-kubernetes#3]
Hi,

I am trying to start the Jobmanager. I have set the context with username password but I am still getting this error message:

```
kubectl` config set-context flink --namespace=flink --cluster=minikube --user=esyekas --password=kashan123`
```

```
kubectl create -f jobmanager-controller.yaml
Please enter Username: esyekas
Please enter Password: kashan123
```
`*********error: You must be logged in to the server (the server has asked for the client to provide credentials)`

Any idea why am I getting this error?

/Kashan

**Comment by melentye** [comment]
Hi Kashan! Let me try to help, can you just double check a few things first:

* if you are using minikube, the `--user` should likely be `minikube` and there is no password, like this: `kubectl config set-context flink --namespace=flink --cluster=minikube --user=minikube` 
* if you are not using minikube, you can check your `~/.kube/config` for user names. 
* don't forget to run `kubectl config use-context flink` after 

Let me know if that helps
/Andrey

**Comment by esyekas** [comment]
Thanks, @melentye . Its worked but when I created just the job manager pods, it gets restarted.

I did step one and two and then I check the pods status:
kubectl get pods
**_No resources found._**
                     
Maybe JM is restarting and not up.

Attachment is the logs for your reference. 
[logs-from-jobmanager-in-jobmanager-controller-xlbwz.txt](https://github.com/melentye/flink-kubernetes/files/1847277/logs-from-jobmanager-in-jobmanager-controller-xlbwz.txt)
